docs/custom-apps.md

```diff
@@ -307,8 +307,6 @@ We want to make it easier to deploy **Zulip Readers** on
 Zulip hardware.  The following document talks about how we want to enable this
 from a code structuring standpoint:
 
-[Writing contrib bots](https://github.com/zulip/zulip/blob/master/contrib_bots/bots/readme.md)
-
```

Comment:
I assume you remove those line because of its outdatedness when it comes to our progress made on `contrib_bots`. Removing this would also require the lines above and below to be edited respectively.

Apart form this, since `contrib_bots` is still under heavy development and still "experimental", I think we should leave the doc in there and update it with our progress made. For now, I think it'd be enough to add a simple 'outdated' on top of it.